rclcpp/src/rclcpp/time_source.cpp

```diff
@@ -190,6 +194,7 @@ void TimeSource::on_parameter_event(const rcl_interfaces::msg::ParameterEvent::S
     if (it.second->value.bool_value) {
       parameter_state_ = SET_TRUE;
       enable_ros_time();
+      create_clock_sub();
```

Comment:
Is this in danger of being called more than once and therefore recreating the topic?

Reply:
for posterity: 9ddb1ae7faf2dbf5405b470cec1f456186284a9d